blockchain/source/contracts/DeployList.sol

```diff
@@ -0,0 +1,37 @@
+pragma solidity ^0.4.23;
+
+
+import "zeppelin-solidity/contracts/ownership/Ownable.sol";
+
+
+contract DeployList is Ownable {
+    address[]deployers;
```

Comment:
the `addresses` is an array of `deployers` or the `deployers` is an array of `addresses`?